spec/models/manageiq/providers/amazon/cloud_manager/metrics_capture_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "spec_helper"
+
+describe ManageIQ::Providers::Amazon::CloudManager::MetricsCapture do
+  before do
+    _guid, _server, @zone = EvmSpecHelper.create_guid_miq_server_zone
+
+    @ems_amazon = FactoryGirl.create(:ems_amazon, :zone => @zone)
+    @vm = FactoryGirl.build(:vm_perf_amazon, :ext_management_system => @ems_amazon)
+  end
+
+  context "With no EMS defined" do
+    it "#perf_collect_metrics raises an error" do
+      expect { @vm.perf_collect_metrics('interval_name') }.to raise_error
+    end
+  end
+
+  context "With an EMS defined" do
```

Comment:
@Fryguy and @chessbyte Rearranging the context and "it" examples titles, I think makes this clearer.

`ManageIQ::Providers::Amazon::CloudManager::MetricsCapture
  #perf_collect_metrics
    handles when nothing is collected
    raises an error when no EMS is defined

Finished in 0.3605 seconds
2 examples, 0 failures`